client/spec/fetch_spec.js

```diff
@@ -77,19 +81,26 @@ describe('Fetch', function() {
         workspace.switchToDesk('SPORTS DESK').then(content.setListView);
         expect(element.all(by.repeater('items._items')).count()).toBe(2);
 
-        browser.get('/#/workspace/ingest');
-        workspace.switchToDesk('SPORTS DESK');
-        content.setListView();
-        content.actionOnItem('Fetch', 0);
-        browser.get('/#/workspace/content');
-        workspace.switchToDesk('SPORTS DESK');
-        content.setListView();
+        browser.get('/#/workspace/ingest').then(function() {
```

Comment:
Based on the documentation from here: 
http://angular.github.io/protractor/#/control-flow
it is not needed to add "then" after every statement